src/runtime_debug.js

```diff
@@ -56,6 +56,14 @@ function isExportedByForceFilesystem(name) {
  * their build, or no symbols that no longer exist.
  */
 function hookGlobalSymbolAccess(sym, func) {
+#if MODULARIZE && !EXPORT_ES6
+  // In MODULARIZE mode the generated code runs inside a function scope and not
+  // the global scope, and JavaScript does not provide access to function scopes
+  // so we cannot dynamically modify the scrope using `defineProperty` in this
+  // case.
+  //
+  // In this mode we simply ignore requests for `hookGlobalSymbolAccess`.
+#else
```

Comment:
To make it clearer why it is valid to ignore here, how about renaming this function to `addWarningOnGlobalSymbol`? By saying that it is a warning, it is clear that it is optional.

Reply:
OK, will do, however all functions in this file are debug only so I think it should fairly clear.

Reply:
Ah, good point. In that case lgtm with just a comment here, like "As everything else in this file, these are just debug checks, so we simply avoid them in this case."